lib/modules/manager/bitbucket-pipelines/extract.ts

```diff
@@ -59,3 +38,43 @@ export function extractPackageFile(content: string): PackageFileContent | null {
   }
   return { deps };
 }
+function addDepAsBitbucketTag(
+  deps: PackageDependency<Record<string, any>>[],
+  pipe: string
+): void {
+  const [depName, currentValue] = pipe.split(':');
+
+  const dep: PackageDependency = {
+    depName,
+    currentValue,
+    datasource: 'bitbucket-tags',
```

Comment:
Use a reference to the id element instead of hardcoding this.

Reply:
Updated